packages/services/f-feature-management/src/FeatureManager.js

```diff
@@ -49,11 +52,16 @@ class FeatureManager {
 
             const context = getContext();
 
+            if (!context) {
+                logger.logError('Unable to provide feature values as no context was required. Check your settings for the plugin');
```

Comment:
This might be my misunderstanding of how this is supposed to work, but 'required' seems the wrong word here? Some suggestions that make more sense to my mind:
```suggestion
                logger.logError('Unable to provide feature values as no context was provided. Check your settings for the plugin');
```
```suggestion
                logger.logError('Unable to provide feature values as no context was given. Check your settings for the plugin');
```
```suggestion
                logger.logError('Unable to provide feature values as no context was obtained. Check your settings for the plugin');
```

Reply:
I think I was going for "required context was not provided" and somehow ended up with this grammatical mess - I like your suggestion of "provided" so let's go with that.